### PR TITLE
Format components fallible and geometry #21373

### PR DESCRIPTION
**Pull request by kingdido999**
```bash
rustfmt components/fallible/*.rs
rustfmt components/geometry/*.rs
```

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21598)

### CODE REVIEW

File: components/geometry/lib.rs [hunk 1]
```diff
@@ -6,7 +6,8 @@ extern crate app_units;
 extern crate euclid;
 extern crate malloc_size_of;
 extern crate style_traits;
-#[macro_use] extern crate malloc_size_of_derive;
+#[macro_use]
+extern crate malloc_size_of_derive;
```
Comment (jdm) [inline components/geometry/lib.rs]: I'm not sure why this got past test-tidy previously, but please move this crate above style_traits.

Comment (kingdido999) [reply components/geometry/lib.rs]: Done.

**Pull request merged**